app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -26,17 +30,35 @@ class MusicsListAdapter(private var onItemClick: (Music) -> Unit) :
 
 class MusicListItemViewHolder(
     private val mBinding: ItemMusicBinding,
-    private val onItemClick: ((Music) -> Unit)
+    private val onItemClickListener: OnClickListener
 ) :
     BaseViewHolder<MusicRepoModel>(mBinding.root) {
 
     override fun onBind(obj: MusicRepoModel) {
         mBinding.txtMusicTitle.text = obj.title
         mBinding.txtArtist.text = obj.artist
-
+        favoriteStateHandler(obj)
         sendToPlay(obj)
     }
 
+    private fun favoriteStateHandler(obj: MusicRepoModel) {
```

Comment:
barayeh in kar bayad ye chek baezary keh agar isfavorite hast che faal bash va age nist ger faal.
bad az har click bayad chek besh va agar isfavorite hast rangesh tagir koneh.(moshkeh avaz nashodan ham hal mishe)